### ISSUE TITLE
Enhance a11y for `Airy`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5410]
name: Airy

### Description
6 accessibility violation was found in the `Airy`.

#### [Components / Airy / Basic](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--basic)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Airy / Duration](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--duration)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Airy / Delay](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--delay)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Airy / Disabled](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--disabled)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Airy / ReadOnly](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--read-only)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

#### [Components / Airy / CustomControl](https://yamada-ui.github.io/yamada-ui/?path=/story/components-airy--custom-control)

Received:
Buttons must have discernible text (button-name)

Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element does not have an implicit (wrapped) <label>
  Element does not have an explicit <label>
  Element's default semantics were not overridden with role="none" or role="presentation"

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
